integration-cli/docker_cli_experimental_test.go

```diff
@@ -0,0 +1,24 @@
+// +build experimental
+
+package main
+
+import (
+	"os/exec"
+	"strings"
+
+	"github.com/go-check/check"
+)
+
+func (s *DockerSuite) TestExperimentalVersion(c *check.C) {
```

Comment:
The thing is I *could* add `-experimental` in my `VERSION` file and get it anyway.